fbsimctl/FBSimulatorControlKit/Sources/Events.swift

```diff
@@ -43,6 +43,7 @@ public enum EventName : String {
   case Uninstall = "uninstall"
   case Upload = "upload"
   case WatchdogOverride = "watchdog_override"
+  case setLocation = "setLocation"
```

Comment:
Camel casing is inconsistent with the naming of the other APIs. Use `set_location` as this matches the `watchdog_override` naming.